### PR TITLE
Remove broken links

### PR DESCRIPTION
**Pull request by jgerigmeyer**
Based on https://github.com/oddbird/oddleventy/actions/runs/6369115715

### CODE REVIEW

File: content/talks/no-grid-system.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ events:
...
-    video: https://vimeo.com/268576559
-    media: &bt
-      iframe: https://player.vimeo.com/video/268576559
-      width: 640
-      height: 360
+    # video: https://vimeo.com/268576559
+    # media: &bt
+    #   iframe: https://player.vimeo.com/video/268576559
+    #   width: 640
+    #   height: 360
```
Comment (jgerigmeyer) [inline content/talks/no-grid-system.md:35]: @mirisuzanne It looks like these are no longer available?

File: content/blog/elsewhere/css-tricks/ethics.md [hunk 2]
```diff
@@ -96,15 +96,13 @@ some resources that I recommend:
...
-    an Atomic Bomb]
+- Mike Monteiro -[How Designers Destroyed the World]
 - Brainstorm Design 2019 -[The Ethics of Design I Fortune]
 - Cathy O'Neil -[Weapons of Math Destruction]
 
 [Book]: https://cennydd.com/future-ethics
 [Talk]: https://www.youtube.com/watch?v=DfxMv9mIjFU&t=381s
 [Code of Ethics]: https://deardesignstudent.com/a-designers-code-of-ethics-f4a88aca9e95
 [How Designers Destroyed the World]: https://vimeo.com/122022963
-[How to Build an Atomic Bomb]: https://vimeo.com/268704084
```
Comment (mirisuzanne) [inline content/blog/elsewhere/css-tricks/ethics.md:108]: https://youtu.be/ZGsYHws-hbc

File: content/talks/no-grid-system.md (hunk 1 above)
Comment (mirisuzanne) [reply content/talks/no-grid-system.md:35]: @jgerigmeyer looks like they moved to youtube

```suggestion
    video: https://youtu.be/iODambgMZhk
    media: &bt
      iframe: https://www.youtube.com/embed/iODambgMZhk
```

File: content/blog/2019/generate.md [hunk 3]
```diff
@@ -116,10 +116,7 @@ Miriam and I had the privilege of watching Jen Simmons live-code a
 fiendishly difficult layout (that we selected for her) using CSS Grid.
 Wow! Such exciting new possibilities with [CSS Grid].
 
-Watch Jen's talk: [Everything You Know About Web Design Just Changed]
-
   [CSS Grid]: https://gridbyexample.com/examples/
-  [Everything You Know About Web Design Just Changed]: https://vimeo.com/316326187
```
Comment (mirisuzanne) [inline content/blog/2019/generate.md:122]: https://www.youtube.com/watch?v=jBwBACbRuGY

**Review by mirisuzanne — APPROVED** [review]
lgtm!

**Pull request merged**